src/codegen/lang/loop.cpp

```diff
@@ -82,8 +82,7 @@ void Loop::CollectFinalLoopVariables(std::vector<llvm::Value *> &loop_vals) {
   PL_ASSERT(last_loop_bb_ != nullptr);
   for (auto *phi_node : phi_nodes_) {
     llvm::PHINode *end_phi =
-        cg_->CreatePHI(phi_node->getType(), 2 + break_bbs_.size(),
-                       phi_node->getName() + "End");
+        cg_->CreatePHI(phi_node->getType(), 2, phi_node->getName() + "End");
```

Comment:
I don't think this is correct.

Reply:
I have to admit I can't remember why I changed this. I will change it back and if it had a reason I might remember when an error occurs.